### DISCUSSION TITLE
Add Series back to Continue Series section

### DISCUSSION THREAD

**Discussion by DarrowLykos** [advplyr/audiobookshelf#1154] [Ideas]
After you remove a series from the Continue Series section, there doesn't seem to be a way of adding it back in. This would be a useful feature to have when you've removed a series in error.

**Comment by advplyr** [comment]
I created an issue for this #1159 and will follow up there

**Reply by advplyr** [reply]
Added in [v2.2.5](https://github.com/advplyr/audiobookshelf/releases/tag/v2.2.5)
